docs/controls/controls.rst

```diff
@@ -0,0 +1,54 @@
+=====================
+Qt Quick Controls 2.0
+=====================
+
+.. sectionauthor:: `e8johan <https://github.com/e8johan>`_
+
+.. github:: ch105
+
+.. note::
+
+    Last Build: |today|
+
+    The source code for this chapter can be found in the `assets folder <../../assets>`_.
+
+This chapter introduces how to use the Qt Quick Controls 2.0 to create a user interface built from standard components such as buttons, labels, sliders and so on. We will look at how various styles can be used to move user interfaces between platforms before diving into custom styling.
+    
+Introduction to Controls
+========================
+
+- What controls exist
+- How do the styles look
+- How to use them on desktop
+- How to use them on devices
```

Comment:
What does this mean? What will the content be?

Reply:
Same here, the contents is supposed to address these questions.

Reply:
Yes, and that's what I'm asking about. :) What will the content be? I'm curious what you plan on putting here, as it affects how I review it.